p2p/src/config.rs

```diff
@@ -13,20 +13,26 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-use serde::{Deserialize, Serialize};
+use utils::make_config_setting;
 
 pub const MDNS_DEFAULT_QUERY_INTERVAL: u64 = 0;
```

Comment:
these constants are duplicated in node/src/config_files/p2p.rs. I think they should be defined once